info/v1/machine.go

```diff
@@ -178,6 +178,9 @@ type MachineInfo struct {
 	// The time of this information point.
 	Timestamp time.Time `json:"timestamp"`
 
+	// Vendor id of CPU.
+	VendorID string `json:"vendor_id"`
```

Comment:
Consider renaming to `CPUVendorID` - it's property of `Machine` but represents CPU vendor.